WalletWasabi.Fluent/ViewModels/Wallets/CoinJoinStateViewModel.cs

```diff
@@ -279,8 +279,9 @@ private void ConfigureStateMachine(CoinJoinManager coinJoinManager)
 
 	private void UpdateCountDown()
 	{
-		ElapsedTime = $"{GetElapsedTime():g}";
-		RemainingTime = $"-{GetRemainingTime():g}";
+		var format = @"hh\:mm\:ss";
+		ElapsedTime = $"{GetElapsedTime().ToString(format)}";
```

Comment:
Can't it be like this:

```suggestion
		ElapsedTime = $"{GetElapsedTime():hh\:mm\:ss}";
```

Though I admit this is a little less readable